### ISSUE TITLE
upgrade matplotlib

### ISSUE THREAD

**Issue by alaniwi** [cedadev/jasmin_scivm#24]
1.4.3 available

**Comment by alaniwi** [comment]
(requested by user)

**Comment by alaniwi** [comment]
Requires dateutil.

**Comment by alaniwi** [comment]
Requires pyparsing.

**Comment by alaniwi** [comment]
Builds `/usr/lib/python2.7/site-packages/matplotlib/mpl-data/matplotlibrc` with `backend : pyside`. pyside is installed, and on import get:

```
/usr/lib/python2.7/site-packages/matplotlib/__init__.py:1039: UserWarning: Bad val "pyside" on line #38
        "backend      : pyside
"
        in file "/usr/lib/python2.7/site-packages/matplotlib/mpl-data/matplotlibrc"
        Key backend: Unrecognized backend string "pyside": valid strings are [u'pgf', u'cairo', u'MacOSX', u'CocoaAgg', u'gdk', u'ps', u'GTKAgg', u'nbAgg', u'GTK', u'Qt5Agg', u'template', u'emf', u'GTK3Cairo', u'GTK3Agg', u'WX', u'Qt4Agg', u'TkAgg', u'agg', u'svg', u'GTKCairo', u'WXAgg', u'WebAgg', u'pdf']
  (val, error_details, msg))
```

pyside is installed (`import PySide`)

Looks like this issue is being looked into and should be fixed in MPL 1.5 (not yet released) - see: https://github.com/matplotlib/matplotlib/issues/4883

For the moment, if edited to say `backend : TkAgg` (as per existing RPM for matplotlib 1.2.0), it works.

spec file hacked to edit matplotlibrc to use TkAgg, resulting in working RPM.
